### PR TITLE
Make Whisper E2E script more portable

### PR DESCRIPTION
**Pull request by kunal-vaishnavi**
This PR makes the Whisper E2E script more portable for other environments.

### CODE REVIEW

File: tutorials/whisper_e2e.py [hunk 1]
```diff
@@ -216,26 +221,31 @@ def postprocessing(token_ids, hf_processor):
...
+        cpu_only=True)
 
-    onnx.save_model(fn_decoder.onnx_model, "whisper_post.onnx")
+    onnx.save_model(fn_decoder.onnx_model, os.path.join(root_dir, "whisper_post.onnx"))
     return fn_decoder(token_ids)
 
 
 if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-a", "--audio", required=True, help="Path to audio file")
```
Comment (wejoncy) [inline tutorials/whisper_e2e.py:233]: add constraint of type=Path?

**Review by wenbingl — APPROVED** [review]
This script was created for a demo initially, and it becomes better now that it can be used as a tool

File: tutorials/whisper_e2e.py [hunk 2]
```diff
@@ -160,30 +163,31 @@ def preprocessing(audio_data):
             "audio_pcm": {1: "sample_len"},
         }
     )
-    onnx.save_model(pre_model, prep_model_name)
+    onnx.save_model(pre_model, os.path.join(root_dir, prep_model_name))
     if USE_ONNX_STFT:
         pre_model = _to_onnx_stft(pre_model)
 
-    pre_f = PyOrtFunction.from_model(pre_model)
+    pre_f = PyOrtFunction.from_model(pre_model, cpu_only=True)
```
Comment (wenbingl) [inline tutorials/whisper_e2e.py:170]: Did you encounter any issue if no cpu_only specified, or just be safe?

Comment (kunal-vaishnavi) [reply tutorials/whisper_e2e.py:170]: If `cpu_only` is not specified, there is a CUDA out of memory error that occurs.

**Pull request merged**